src/handlers/dns/dns.cpp

```diff
@@ -18,7 +18,9 @@ AggDomainResult aggregateDomain(const std::string &domain)
         return AggDomainResult(qname2, qname3);
     }
     std::size_t endDot = std::string::npos;
-    if (domain.back() == '.') {
+    if (!static_suffix.empty() && domain.size() > static_suffix.size()) {
```

Comment:
if this doesn't need to compare the suffix (because it's already compared I guess?), why not just pass a size_t instead?

Reply:
it is already compared in the filter. so I think we should not compare it again. It is better to use size_t only